README.md

```diff
@@ -109,6 +109,8 @@ randomly_terminate_test_length:   # amount of time, in minutes, to run test_SR_I
                                   # example: 10.5
 container_manager:                # the container manager command to use (podman or docker)
                                   # example: podman
+container_volumes:                # the volumes to use with the container command
```

Comment:
in the comment clarify it is "volume mapping"

Reply:
Addressed in most recent commit